etna/packages/etna-js/upload/actions/upload_actions.jsx

```diff
@@ -70,70 +59,20 @@ export const fileSelected = ({ file, folder_name, bucket_name }) => (dispatch, g
   }
 }
 
-export const startUploadActions = ({
-  file,
-  file_name,
-  url,
-  dispatch,
-  getState
-}) => {
-  dispatch({
-    type: ADD_UPLOAD,
-    project_name: CONFIG.project_name,
-    file,
-    file_name,
-    url
-  });
-  let upload = getUpload(getState(), file_name);
-
-  work(dispatch, "start", { upload });
-};
-
-export const uploadStarted = ({ file_name }) => (dispatch, getState) => {
-  let upload = getUpload(getState(), file_name);
-
-  if (upload.status == 'active') work(dispatch, 'continue', { upload });
-}
-
-export const uploadBlobCompleted = ({ file_name }) => (dispatch, getState) => {
-  let upload = getUpload(getState(), file_name);
-
-  if (upload.status == 'active') work(dispatch, 'continue', { upload });
+export const unpauseUpload = ({ upload }) => (dispatch) => {
+  dispatchUploadWork(dispatch, new UnpauseUploadCommand(upload));
```

Comment:
is this use of new required? You don't use it below...

Reply:
Oh, no, it's not, let me adjust.  Originally they were classes but I switched back to just plain old functions.

Reply:
Incidentally, due to weird javascript semantics, a new on a function that returns an instance acts the same as that function invocation without the new...